docs/app/Examples/elements/Header/Types/HeaderSubHeadersExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderSubHeadersExamples extends Component {
+  render() {
+    return (
+      <Header.H2 className='sub'>
```

Comment:
Yeah, weirdly enough it needs that. That is what makes the header text display small. This one is a very specific example.